src/Main/java/Commands/AskWatsonCommand.java

```diff
@@ -5,6 +5,11 @@
 public class AskWatsonCommand implements Command {
     @Override
     public void execute(String[] args, GameContext context) {
+        if (!context.isCaseStarted()) {
+            System.out.println("The case has not started yet. Type 'start case' to begin the investigation.");
+            return;
+        }
+
```

Comment:
maybe factor this out to a common place? make an abstract command class that implements the logic of checking if the game has started